account_banking_payment_export/README.rst

```diff
@@ -76,6 +76,7 @@ Contributors
 * Erwin van der Ploeg
 * Raphaël Valyi
 * Sandy Carter
+* Angel Moya <angel.moya@domatix.com>
```

Comment:
I can remove on both because it's a little contribution, put note field  it's on account_banking_payment_export module because this module is the base, and I think this field could be usefull in other modules.